### PR TITLE
Add a github release automation script

### PR DESCRIPTION
**Pull request by zijun726911**
### Changes
- Add a github release automation script (use the steps in our internal release doc as reference)

### Test
Run this script from my local ` RELEASE_VERSION=v1.0.2  ./scripts/github-release.sh`, using this script did the v1.0.2 github release, it's able to push release branch, tag and send a PR 

In the future, probably this script can be triggered by github action and run in github action worker instance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by mikhail-aws — APPROVED** [review]

File: scripts/github-release.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
```
Comment (mikhail-aws) [inline scripts/github-release.sh:1]: bin/bash

File: scripts/github-release.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "$WORKSPACE_DIR/examples/deploy-$RELEASE_VERSION.yaml" \
+  "$WORKSPACE_DIR/examples/deploy-$VERSION_TO_REMOVE.yaml" \
+  "$WORKSPACE_DIR/docs/guides/deploy.md"
+git commit -m "Release artifacts for release $RELEASE_VERSION"
+git push origin release-$RELEASE_VERSION
+git tag -a  $RELEASE_VERSION -m "Release artifacts for release $RELEASE_VERSION"
+git push origin $RELEASE_VERSION
+
+# Create a PR
+gh pr create --title "Release artifacts for $RELEASE_VERSION" --body "Release artifacts for $RELEASE_VERSION" \
```
Comment (mikhail-aws) [inline scripts/github-release.sh:72]: gh? we should not enforce gh cli usage

**Review by mikhail-aws — CHANGES_REQUESTED** [review]

File: scripts/github-release.sh (hunk 1 above)
Comment (zijun726911) [reply scripts/github-release.sh:1]: will change

File: scripts/github-release.sh (hunk 2 above)
Comment (zijun726911) [reply scripts/github-release.sh:72]: I referenced the: https://github.com/aws/amazon-vpc-cni-k8s/blob/4dac30bddbc2bd5970737b9c000159c9a8e69dfb/scripts/sync-to-eks-charts.sh#L193

just send a PR to automate release process? what do you think?

**Review by mikhail-aws — APPROVED** [review]

**Pull request merged**